src/platform/packages/shared/kbn-actions-types/action_types.ts

```diff
@@ -24,4 +24,5 @@ export interface ActionType {
   supportedFeatureIds: string[];
   isSystemActionType: boolean;
   subFeature?: SubFeature;
+  isDeprecated: boolean;
```

Comment:
I'm curious why we are adding this as non-optional.  I believe you mentioned it in a team chat the other day, but now I don't remember the reason :-)

Reply:
Because we always populate it, It never becomes `undefined` always either `true` or `false`

Reply:
Right, but we could have made it optional, where `undefined` was treated as false, which is a pretty standard pattern.  It would have caused a lot fewer files to be touched (mainly tests).  

So curious why we chose to do it this way, rather than make it optional.

Reply:
It just sounds semantically wrong.
I discussed this with @darnautov  and we decided not to make something required optional just to create less changes. And that is why we wanted to add the mock creator function and the jest matchers. Next time we need something required, it will not create that much changes :)